### DISCUSSION TITLE
Decode tasmota information

### DISCUSSION THREAD

**Discussion by ortegafernando** [arendst/Tasmota#15857] [Q&A] [answered]
Hi, I want to know if there is any action to know "ALL" tasmota configuration. so I could check my tasmota as backup data.
I have seen in status3 things like Resolution, setoption, status4 Features, ...

Is there any help page or any rule to "decode" this json data? 

Thanks

**Comment by sfromis** [comment, marked as answer]
For all config data, use the tool https://github.com/tasmota/decode-config/
Allows you to get it in command format (likely the most practical), or JSON format.

Another tool is https://github.com/tasmota/decode-status/ which decodes some of the Status messages, but far less than decode-config.
